dist/systemd/barrierc@.service.in

```diff
@@ -0,0 +1,59 @@
+#  Barrier Client service barrierc-@BARRIER_VERSION@
+#
+#  This file is part of Barrier.
+#
+#  This is a systemd template service for the barrierc client service. Instances
+#  are started using @<hostname>. For example:
+#
+#  Starting:
+#   systemctl start barrierc@myserver
+#  or:
+#   systemctl start barrierc@myserver:24800
+#
+#  Enabling:
+#   systemctl enable barrierc@myserver
+#
+#  SSL Fingerprints are stored in /var/db/barrier@<hostname> for each client 
```

Comment:
I think it would be better to use a single directory for all barrier data, e.g. `/var/lib/barrier/hostname-<hostname>`.

Reply:
It could use a single directory, but that makes it impossible to automatically add the fingerprint for a new server on the first run if there isn't one already. The only way I found that works (within the scope of what systemd can do without writing a more lengthy external script) is checking if there is a `TrustedServers.txt` file. 

The only other way that I could think of would be automatically appending a fingerprint with `>>` instead, which has some security concerns.